acrobat/scripts/ccTranslations.js

```diff
@@ -0,0 +1,33 @@
+export default function ccTranslations() {
+  const pTags = document.documentElement.querySelectorAll('p[data-local="credit-cards"]');
+  function checkIfImageExists(url) {
+    const http = new XMLHttpRequest();
+    http.open('HEAD', url, false);
+    http.send();
+    if (http.status !== 404) {
+      return true;
+    }
+    return false;
+  }
+  function errorLogging() {
+    window.lana?.log('No image available for credit-card placeholder');
+  }
+
+  pTags.forEach((tag) => {
+    const ccimg = document.createElement('img');
```

Comment:
MILO has method for to create elements easier

Reply:
+1 it is called createTag()